xbar/xbar.h

```diff
@@ -1,7 +1,7 @@
 #ifndef __XBAR_H
 #define __XBAR_H
 
-#include <strings.h>
+#include "strings.h"
```

Comment:
I would favor this instead:

```c++
#ifndef _WIN32
#include <strings.h>
#else
#include "strings.h"
#endif
```

Or to add `contrib/` as an include folder for Windows builds